schema/vrs.yaml

```diff
@@ -64,17 +64,18 @@ definitions:
     - state
     - type
   Haplotype:
-    description: A set of non-overlapping Alleles that co-occur on the same molecule.
+    description: A set of non-overlapping Allele members that co-occur on the same
+      molecule.
     additionalProperties: false
     type: object
     properties:
-      _id:
-        $ref: '#/definitions/CURIE'
+      _id: *id001
```

Comment:
Why use refs only here? We should use consistently.

Reply:
This is only done here because I've only added Haplotype and Allele to inherit from Variation so far. As I add others, inherited properties (really, just `_id`) will be used by ref in all applicable classes.

Of course, this is just a YAML convention and in JSON Schema, each instance will remain fully described.